### DISCUSSION TITLE
I`m started the project (npm start)

### DISCUSSION THREAD

**Discussion by RomarioDeveloper** [magicbell/community#162] [Questions] [answered]
![image](https://github.com/magicbell-io/community/assets/85926208/c7a9e9f0-e163-43b0-93a8-7cb44ffe2dd5)

how to fix that

**Comment by RomarioDeveloper** [comment]
![image](https://github.com/magicbell-io/community/assets/85926208/d9f1e82d-86c3-489e-a014-aaba6dc10960)
npm run dev

**Reply by unamashana** [reply, marked as answer]
I am not sure if you are asking about testing, but if you are, you'd need to run this over a https tunnel for it to work.
